### PR TITLE
feedback from review

### PR DESCRIPTION
**Pull request by malaffoon**

### CODE REVIEW

File: docs/API-Ingest.md [hunk 1]
```diff
@@ -267,11 +267,7 @@ curl -X POST --header "Authorization:Bearer {access_token}" -F file=@winterICA.1
 ```
   
 #### Resubmit Exams
-The system accepts and stores import payloads regardless of processing status. For imports with certain problems like 
-an unknown school or assessment, the import may be "replayed" once the underlying cause is resolved. For example, if
-the missing school is added to the system, all affected exams can be resubmitted, without requiring the client to 
-re-post the data. Note that this is meant as an operational utility; if it is being used regularly there are probably
-fundamental data flow issues. 
+The system accepts and stores import payloads regardless of processing status. For imports with certain problems like an unknown school or assessment, the import may be "replayed" once the underlying cause is resolved. For example, if the missing school is added to the system, all affected exams can be resubmitted, without requiring the client to re-post the data. Note that this is meant as an operational utility; if it is being used regularly there are probably fundamental data flow issues.
```
Comment (malaffoon) [inline docs/API-Ingest.md]: reformatting only (newlines)

File: docs/API-Ingest.md [hunk 2]
```diff
@@ -303,10 +299,7 @@ curl -X POST --header "Authorization:Bearer {access_token}" https://import-servi
 End-points for submitting organization data; this includes districts, schools, and groups of institutions. 
 
 #### Create Organization Import Request
-Accepts JSON payload, compatible with the format produced by ART. The payload should contain all the organization
-data necessary to resolve all contents; for example, if a school is in a group under a district, all three must
-be present in the payload. The payload must be valid JSON but the exact structure doesn't matter a lot: the system
-will parse the payload looking for the required fields: `entityType`, `entityId`, `entityName`, `parentEntityId`. 
+Accepts JSON payload, compatible with the format produced by ART. The payload should contain all the organization data necessary to resolve all contents; for example, if a school is in a group under a district, all three must be present in the payload. The payload must be valid JSON but the exact structure doesn't matter a lot: the system will parse the payload looking for the required fields: `entityType`, `entityId`, `entityName`, `parentEntityId`.
```
Comment (malaffoon) [inline docs/API-Ingest.md]: reformatting only (newlines)

File: docs/Monitoring.md [hunk 3]
```diff
@@ -6,9 +6,18 @@
 
 Monitoring RDW applications includes monitoring:
 
-* Database. 
-* Logging.
-* Application Status.
+* [Database](#database)
```
Comment (malaffoon) [inline docs/Monitoring.md]: oh, i need to test these links and add anchors if necessary

File: docs/Monitoring.md [hunk 4]
```diff
@@ -96,6 +105,9 @@ GROUP BY delay, bucket
 ORDER BY bucket DESC, delay;
 ```
 
+#### Validate Migrate Data Integrity
+There is a script that reports on data discrepancies between the data warehouse and the reporting data mart(s). Detailed instructions for running that scripts and interpreting the results can be found in [RDW_Schema](https://github.com/SmarterApp/RDW_Schema) under the `validation` folder.
+
```
Comment (agorina) [inline docs/Monitoring.md]: I liked how you phrased it without `develop`. I will update the docs I created to use this approach.

**Pull request merged**